### ISSUE TITLE
Failed creating new buffalo app in Debian WSL

### ISSUE THREAD

**Issue by karfianto** [gobuffalo/cli#46]
### Description
I've tried creating new project with buffalo but the generated files are not complete. I've read some related issues, check my GOPATH, update to latest Go, but still unable to get it done.

### Steps to Reproduce the Problem

```zsh
> pwd
/mnt/c/Users/intruder/go/src/github.com/karfianto

> buffalo new mypro --db-type mysql --ci-provider none --vcs none --skip-config
> cd mypro
> ls -la
total 192
drwxrwxrwx 1 sysadmin sysadmin   4096 Nov 13 23:31 .
drwxrwxrwx 1 sysadmin sysadmin   4096 Nov 13 23:30 ..
drwxrwxrwx 1 sysadmin sysadmin   4096 Nov 13 23:30 config
-rwxrwxrwx 1 sysadmin sysadmin   1432 Nov 13 23:30 database.yml
-rwxrwxrwx 1 sysadmin sysadmin   4457 Nov 13 23:31 go.mod
-rwxrwxrwx 1 sysadmin sysadmin 182529 Nov 13 23:31 go.sum
drwxrwxrwx 1 sysadmin sysadmin   4096 Nov 13 23:30 grifts
drwxrwxrwx 1 sysadmin sysadmin   4096 Nov 13 23:30 models
drwxrwxrwx 1 sysadmin sysadmin   4096 Nov 13 23:31 node_modules
-rwxrwxrwx 1 sysadmin sysadmin     86 Nov 13 23:31 yarn.lock 
```
### Expected Behavior

Should be able to run `gobuffalo dev`

### Actual Behavior

Error due to incomplete files.

### Info
```zsh
> buffalo info
-> Go: Checking installation
✓ The `go` executable was found on your system at: /usr/local/go/bin/go

-> Go: Checking minimum version requirements
✓ Your version of Go, 1.17.3, meets the minimum requirements.

-> Go: Checking Package Management
✓ You are using Go Modules (`go`) for package management.

-> Go: Checking PATH
✓ Your PATH contains /mnt/c/Users/intruder/go/bin.

-> Node: Checking installation
✓ The `node` executable was found on your system at: /usr/bin/node

-> Node: Checking minimum version requirements
✓ Your version of Node, v14.18.1, meets the minimum requirements.

-> NPM: Checking installation
✓ The `npm` executable was found on your system at: /usr/bin/npm

-> NPM: Checking minimum version requirements
✓ Your version of NPM, 6.14.15, meets the minimum requirements.

-> Yarn: Checking installation
✓ The `yarnpkg` executable was found on your system at: /usr/bin/yarnpkg

-> Yarn: Checking minimum version requirements
✓ Your version of Yarn, 1.22.17, meets the minimum requirements.

-> PostgreSQL: Checking installation
✘ The `postgres` executable could not be found on your system.
For help setting up your Postgres environment please follow the instructions for you platform at:

https://www.postgresql.org/download/

-> MySQL: Checking installation
✓ The `mysql` executable was found on your system at: /usr/bin/mysql

-> MySQL: Checking minimum version requirements
✓ Your version of MySQL, 15.1, meets the minimum requirements.

-> SQLite3: Checking installation
✓ The `sqlite3` executable was found on your system at: /usr/bin/sqlite3

-> SQLite3: Checking minimum version requirements
✓ Your version of SQLite3, 3.27.2, meets the minimum requirements.

-> Cockroach: Checking installation
✓ The `cockroach` executable was found on your system at: /usr/local/bin/cockroach

-> Cockroach: Checking minimum version requirements
✓ Your version of Cockroach, 19.2.4, meets the minimum requirements.

-> Buffalo (CLI): Checking installation
✓ The `buffalo` executable was found on your system at: /usr/local/bin/buffalo

-> Buffalo (CLI): Checking minimum version requirements
✓ Your version of Buffalo (CLI), v0.17.6, meets the minimum requirements.

-> Buffalo: Application Details
Pwd         /mnt/c/Users/intruder/go/src/github.com/karfianto
Root        /mnt/c/Users/intruder/go/src/github.com/karfianto
GoPath      /mnt/c/Users/intruder/go
PackagePkg  github.com/karfianto
ActionsPkg  github.com/karfianto/actions
ModelsPkg   github.com/karfianto/models
GriftsPkg   github.com/karfianto/grifts
WithModules true
Name        karfianto
Bin         bin/karfianto
VCS
WithPop     false
WithSQLite  false
WithDep     false
WithWebpack false
WithNodeJs  false
WithYarn    false
WithDocker  false
WithGrifts  false
AsWeb       true
AsAPI       false
InApp       false
PackageJSON {map[]}
```

**Comment by ernsheong** [comment]
Same here:

```
jonathan@mycomputer:~/repos$ buffalo new backend
Usage:
  buffalo new [name] [flags]

Flags:
      --api                  skip all front-end code and configure for an API server
      --ci-provider string   specify the type of ci file you would like buffalo to generate [none, travis, gitlab-ci, circleci] (default "none")
      --config string        config file (default is $HOME/.buffalo.yaml)
      --db-type string       specify the type of database you want to use [cockroach, mariadb, mysql, postgres] (default "postgres")
      --docker string        specify the type of Docker file to generate [none, multi, standard] (default "multi")
  -d, --dry-run              dry run
  -f, --force                delete and remake if the app already exists
  -h, --help                 help for new
      --module string        specify the root module (package) name. [defaults to 'automatic']
      --skip-config          skips using the config file
      --skip-pop             skips adding pop/soda to your app
      --skip-webpack         skips adding Webpack to your app
      --skip-yarn            use npm instead of yarn for frontend dependencies management
      --vcs string           specify the Version control system you would like to use [none, git, bzr] (default "git")
  -v, --verbose              verbosely print out the go get commands

ERRO[0000] Error: stat /home/jonathan/repos/vcs/templates/ignore.tmpl: no such file or directory
```

Error: 
`ERRO[0000] Error: stat /home/jonathan/repos/vcs/templates/ignore.tmpl: no such file or directory`

**Comment by fasmat** [comment]
> Same here, buffalo **v0.17.6**, WSL Ubuntu:

A new version **v0.17.7** was released a few hours ago. It fixes this issue present in v0.17.6
